students/BrianE/session04/trigrams.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+"""
+Trigrams Lab
+"""
+
+import random
+
+
+def import_text(file):
+    """
+    Import text from file and return a list of words.
+    :param file: input file containing starting text
+    :return: list of words
+    """
+
+    # Only use alpha-numeric words from file
+    with open(file=file, mode='r') as text:
+        word_list = [word for word in text.read().split() if word.isalnum()]
```

Comment:
what is this going to do for contractions: can't or parentheses: (this is in parens) (would "think" and "parens" get used?